README.md

```diff
@@ -2,12 +2,13 @@
 
 [![Build Status](https://travis-ci.org/ayojs/ayo.svg?style=flat&branch=master)](https://travis-ci.org/ayojs/ayo?branch=master)
 
-_(Note: Ayo.js is forked from [Node.js][]. Currently, a lot of the documentation
-still points towards the Node.js repository.)_
+(Note: Ayo.js is forked from [Node.js][]. Currently, a lot of the documentation
+still points towards the Node.js repository.)
 
-Ayo.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Ayo.js
+Ayo.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. It
 uses an event-driven, non-blocking I/O model that makes it lightweight and
-efficient. The Ayo.js package ecosystem, [npm][], is the largest ecosystem of
+efficient. Ayo.js shares with the rest of the free JavaScript
```

Comment:
why "free"? are there any that are not free? if not, why distinguish?

Reply:
Well, JavaScript is a language and as such can have any kind of implementation, including non-free. If I remember correctly, ActionScript was actually a non-free implementation of JavaScript for Adobe Flash. But this is probably not the place of time fo make this kind of distinctions, so I'll just remove that. Thanks for the suggestion.